build/templates/_library_interpreter.py/_get_error_description.py.mako

```diff
@@ -13,11 +13,12 @@
         Returns the error description.
         '''
         try:
-            _, error_string = self.get_error(${get_error_params_snippet})
-            return error_string
+            returned_error_code, error_string = self.get_error(${get_error_params_snippet})
+            if returned_error_code == error_code:
+                return error_string
```

Comment:
Why would returned_error_code be different? Won't we raise if an error occurs during the get_error() call?

Reply:
For the local LibraryInterpreter it won't be different, because we're running on the same thread, etc. For gRPC it could be different, and the code here is now shared with both.

(But also, since get_error returns the error_code regardless, if for some odd reason it gave us a different error code, we shouldn't return the wrong string.)